.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+  - "2.7"
+install:
+  - python setup.py install
```

Comment:
Does this actually work? I haven't been able to get it working on my workstation. It complains about some package version mismatch, but maybe travis handles that somehow?

Reply:
Hmm which package is failing? It works inside a clean virtual environment on my workstation and the travis build for my forked repo.

Reply:
Interesting. Looks like it's fine on travis then. I'll figure out why my instances didnt work later.